packages/flutter_markdown/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: A Markdown renderer for Flutter. Create rich text output,
   formatted with simple Markdown tags.
 repository: https://github.com/flutter/packages/tree/main/packages/flutter_markdown
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+flutter_markdown%22
-version: 0.6.14
+version: 0.6.14+1
```

Comment:
I'm curious why this is marked as a `+1` as opposed to `0.6.15`. 

I feel this should be `0.6.15` as this isn't just a bug fix, but I'm happy to be corrected.

Reply:
The CHANGELOG style says that the `update-release-info` command should be run with `--version=minor` for features and with `--version=minimal` for everything else most of the times. Since I'm not adding a feature I decide to run the command with `--version=minimal`.